index.html

```diff
@@ -1,167 +1,106 @@
-            </svg>
-            Support Server
-          </button>
-        </a>
-        <a href="https://docs.ezar.shamdev.xyz">
-          <button type="button" class="btn btn-outline-secondary btn-lg">
-            <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="currentColor" class="bi bi-book-half" viewBox="0 0 16 16">
-              <path d="M8.5 2.687c.654-.689 1.782-.886 3.112-.752 1.234.124 2.503.523 3.388.893v9.923c-.918-.35-2.107-.692-3.287-.81-1.094-.111-2.278-.039-3.213.492V2.687zM8 1.783C7.015.936 5.587.81 4.287.94c-1.514.153-3.042.672-3.994 1.105A.5.5 0 0 0 0 2.5v11a.5.5 0 0 0 .707.455c.882-.4 2.303-.881 3.68-1.02 1.409-.142 2.59.087 3.223.877a.5.5 0 0 0 .78 0c.633-.79 1.814-1.019 3.222-.877 1.378.139 2.8.62 3.681 1.02A.5.5 0 0 0 16 13.5v-11a.5.5 0 0 0-.293-.455c-.952-.433-2.48-.952-3.994-1.105C10.413.809 8.985.936 8 1.783z"/>
-            </svg>
-            Documentation
-          </button>
-        </a>
-      </div>
-      <div class="row">
-        <div class="col-sm-4">
-          <h3>
-            <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="currentColor" class="bi bi-collection-fill" viewBox="0 0 16 16">
-              <path d="M0 13a1.5 1.5 0 0 0 1.5 1.5h13A1.5 1.5 0 0 0 16 13V6a1.5 1.5 0 0 0-1.5-1.5h-13A1.5 1.5 0 0 0 0 6v7zM2 3a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 0-1h-11A.5.5 0 0 0 2 3zm2-2a.5.5 0 0 0 .5.5h7a.5.5 0 0 0 0-1h-7A.5.5 0 0 0 4 1z"/>
-            </svg>
-            Multi Purpose
-          </h3>
-          <p>
-            A multi-purpose bot ready to<br>skill up and boost up your Discord
-            server!
-          </p>
+    <div class="ez-desc">
+        <h1 class="text-5xl font-bold">eZaR Features</h1>
+        <p class="py-6">Here, You will find many features of eZaR.</p>
+    </div>
+
+    <div class="ez-row"> 
+        <div class="ft-zero">
+                <div class="card w-96 bg-primary text-primary-content">
+                    <div class="card-body">
+                    <h2 class="card-title">Constant Updates 🕗</h2>
+                    <p>With our contributors, Our bot gets updated almost everyday staying ahead of time.</p>
```

Comment:
We can't exactly guarentee this, so maybe change it to something like:
```suggestion
                    <p>eZaR recieves updates very often to stay up to date with Discord and fix any bugs that are encountered!</p>
``` This sentence could be re-worded too but it gives a bit of an idea.